dse/cluster.py

```diff
@@ -0,0 +1,24 @@
+from cassandra.cluster import Cluster, Session
+import dse.cqltypes  # unsued here, imported to cause type registration
+from dse.util import Point, Circle, LineString, Polygon
+
+
+class Cluster(Cluster):
+
+    def _new_session(self):
```

Comment:
Does `_new_session()` need to be reimplemented in DSE cluster? The code looks the same.